### PR TITLE
i18n(zh-cn): update `guides/styling.mdx`

### PR DESCRIPTION
**Pull request by dorasans**
#### What kind of changes does this PR include?
- Do some minor content fixes,translate the part of  `Production`.

#### Description

- update `guides/styling.mdx`

### CODE REVIEW

**Comment by dorasans** [comment]
@liruifengv pls review

**Comment by liruifengv** [comment]
PR 描述分类改一下

File: src/content/docs/zh-cn/guides/styling.mdx [hunk 1]
```diff
@@ -507,10 +521,34 @@ Astro 中的 Svelte 也完全按照预期工作。[Svelte Styling Docs][svelte-s
 
 如果你使用 Tailwind，[typography 插件](https://tailwindcss.com/docs/typography-plugin) 可以用于样式化 Markdown。
 
+## 生产
+
+### 捆绑包控制
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/styling.mdx:526]:
```suggestion
### 打包控制
```

File: src/content/docs/zh-cn/guides/styling.mdx [hunk 2]
```diff
@@ -527,7 +565,7 @@ import rawStylesCSS from '../styles/main.css?raw';
 <style is:inline set:html={rawStylesCSS}></style>
 ```
 
-完整的细节见[Vite的文档](https://vitejs.dev/guide/assets.html#importing-asset-as-url)。
+完整的细节见[Vite的文档](https://vitejs.dev/guide/assets.html#importing-asset-as-string)。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/styling.mdx:568]:
```suggestion
完整的细节见[Vite的文档](https://cn.vitejs.dev/guide/assets.html#importing-asset-as-string)。
```
Vite 的文档可以指向中文

File: src/content/docs/zh-cn/guides/styling.mdx [hunk 3]
```diff
@@ -464,7 +478,7 @@ npm install less
 
 ## PostCSS
 
-Astro 内置了 PostCSS，作为 [Vite](https://vitejs.dev/guide/features.html#postcss) 的一部分。要为你的项目配置 PostCSS，在项目根部创建 `postcss.config.js` 文件。然后你就你可以在安装插件后使用 `require()` 导入插件（例如 `npm i autoprefixer`）。
+Astro 内置了 PostCSS，作为 [Vite](https://vitejs.dev/guide/features.html#postcss) 的一部分。要为你的项目配置 PostCSS，在项目根部创建 `postcss.config.js` 文件。然后你就可以在安装插件后使用 `require()` 导入插件（例如 `npm install autoprefixer`）。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/styling.mdx:481]:
```suggestion
Astro 内置了 PostCSS，作为 [Vite](https://cn.vitejs.dev/guide/features.html#postcss) 的一部分。要为你的项目配置 PostCSS，在项目根部创建 `postcss.config.js` 文件。然后你就可以在安装插件后使用 `require()` 导入插件（例如 `npm install autoprefixer`）。
```

**Review by liruifengv — CHANGES_REQUESTED** [review]
A little change. Other LGTM! Thanks @dorasans

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🚀

**Pull request merged**